mqtts/Dockerfile

```diff
@@ -6,24 +6,15 @@
 
 # passing arguments to map letsencrypt certs
```

Comment:
This comment looks wrong because you removed lets encrypt.

Reply:
Yes. I removed it.